index.html

```diff
@@ -0,0 +1,263 @@
+            <a href="#"> My Logo </a>
+        </div>
+            <ul class="nav-list">
+                <li class="list-item">
+                    <a href="#"> Portfolio </a>
+                </li>
+                <li class="list-item">
+                    <a href="#"> About </a>
+                </li>
+                <li class="list-item">
+                    <a href="#"> Contact </a>
+                </li>
+            </ul>
+        <div class="menu">
+            <div class="menu-line"></div>
+            <div class="menu-line"></div>
+            <div class="menu-line"></div>
+        </div>
+  </nav>
+ </header>
+
+  <section class="head-line">
+        <div class="container">
+            <div class="headline-text">
+                <h1 class="first-text">I’m Ellis<br> Glad to see you!</h1>
+                 <p>I'm a software developer! i can help you build a product,feature or website look through
+                  some of my work and expeirience! if you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+                 <a href="#" class="connect"><h6>Let's connect</h6></a>
+                <ul class="medium">
+                    <li class="social-media"><a href="#"><i class="fa fa-twitter"></i></a></li>
+                    <li class="social-media"><a href="#"><i class="fa fa-linkedin"></i></a></li>
+                    <li class="social-media"><a href="#"><i class="fa fa-medium"></i></a></li>
+                    <li class="social-media"><a href="#"><i class="fa fa-github"></i></a></li>
+                    <li class="social-media"><a href="#"><i class="fa fa-angellist"></i></a></li>
+                </ul>
+            </div>
+      </div>
+  </section>
+
+  <section class="section1">
+          <ul class="section-wrap">
+               <li class="work">            
+                  <div class="work-container">
+                    <ul>
+                      <li>
+                          <img src="image/portfolio1.png" alt="Several hands holding beer glasses">
+                      <li class="work-description">
+                        <h1><a href="">Tonic</a></h1>
+                        <div class="work-info">
+                                <ul>
+                                  <li class="intro">Canopy</li>
+                                  <li> <i class="fa fa-circle"></i> Back End Dev</li>
+                                  <li><i class="fa fa-circle"></i> 2015</li>
+                                </ul>
+                                </div>
+                        <p>
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                        </p>
+
+                        <div class="tags">
+                          <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                          </ul>
```

Comment:
- [ ] I recommend you to use ul and li only for project technologies, please modify your code to keep to semantic code like section and include your heading and paragraph

-------------------Original ------------------
![design1](https://user-images.githubusercontent.com/55635977/126050346-7dfc05a9-df30-4f45-8493-10a352e1b0ac.png)
-------------------Yours ------------------
  
![design2](https://user-images.githubusercontent.com/55635977/126050359-d0ebfe9b-d57a-4732-9f3b-7544a4c2e089.png)

Kindly, refer to the figma to fix that issue, arrange your project technologies as flex not block or listed.